### ISSUE TITLE
U7 Intro - Speech is cut off in new German translation

### ISSUE THREAD

**Issue by ungesundes-halbwissen** [exult/exult#553]
The new intro speech of the Guardian in my German transslation is longer than the original one. While that is no problem in DOSBOX, Exult cuts it off after the original time slot has ended.

Check here: https://youtu.be/xr2hXNGWZ6Q?t=174
Guardian starts talking right away at 3:07 and will be cut off at 3:57

Any chance to replicate the DOSBOX behaviour and wait for the sound file to have ended?

**Comment by wench** [comment]
Yes this can be changed but waiting for speech to finish will cause music to become out of sync

**Comment by ungesundes-halbwissen** [comment]
That'd be great. While you are right regarding the music ofc, it is the smaller loss in my opinion. Here is the the full speech in DOSBOX: https://www.youtube.com/watch?v=lCkyVj9PjLU Approx duration 1:02

**Comment by wench** [comment]
Guessing should just let the speech to continue to play on the scene change

**Comment by wench** [comment]
I can't access the site https://sirjohn.de/ to download the mod. Can you attach the mod here

**Comment by ungesundes-halbwissen** [comment]
Yeah, .de is offline since the weekend. Please use https://sirjohn.net/

**Comment by wench** [comment]
Ok, got it. I'll improve things

**Comment by wench** [comment]
The fix for this will wait for the speech to finish before the Guardian disappears. It seems very similar to the video of Dosbox
